src/app/api/delete-quiz/route.ts

```diff
@@ -0,0 +1,16 @@
+import {deleteQuizService} from '@/lib/core/service'
```

Comment:
From our previous conversation, I think we will be using REST methods for the API, eg:

GET /api/quiz/123
POST /api/quiz
PUT /api/quiz/123
DELETE /api/quiz/123

Reference: https://www.restapitutorial.com/lessons/httpmethods.html